### PR TITLE
Fix the auto update for Windows

### PR DESCRIPTION
**Pull request by valib**
caused by the commit e0ab226. This bug is reported in https://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=14142

### CODE REVIEW

**Comment by valib** [comment]
@SubJunk this should be relased as soon as possible

File: src/main/java/net/pms/update/AutoUpdaterServerProperties.java [hunk 1]
```diff
@@ -46,6 +46,12 @@ private String getStringWithDefault(String key, String defaultValue) {
 	}
 
 	private String getPlatformSpecificKey(String key) {
-		return key + "." + operatingSystem.toString();
+		String os = operatingSystem.toString();
+		 //we use only the one UMS Windows version so the exact OS Windows version is not relevant
+		if (os.startsWith("windows")){
+			os = "windows";
+		}
```
Comment (SubJunk) [inline src/main/java/net/pms/update/AutoUpdaterServerProperties.java:53]: This should make it work on macOS too

```suggestion
		if (!os.startsWith("linux")) {
			os = operatingSystem.getPlatformName();
		}
```

**Comment by SubJunk** [comment]
@valib thanks for the fix, I've modified it to also fix macOS. Please let me know if you approve of that change, then I will merge if you do

**Comment by valib** [comment]
@SubJunk I agree

**Pull request merged**